core/services/vrf/helpers_test.go

```diff
@@ -1,7 +1,32 @@
 package vrf
 
-import "math/big"
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
 
 func GenerateProofWithNonce(secretKey, seed, nonce *big.Int) (*Proof, error) {
 	return generateProofWithNonce(secretKey, seed, nonce)
 }
+
+func GenerateProofResponseWithNonce(secretKey *big.Int, preSeed *big.Int,
```

Comment:
Again, I wonder if some sort of nicely named struct would be more appropriate for an argument here

Reply:
Thanks.